src/BenchmarksApps/HttpClientBenchmarks/Clients/HttpClient/Program.cs

```diff
@@ -0,0 +1,254 @@
+﻿using System.Collections.Concurrent;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.Diagnostics;
+using System.Net.Security;
+using System.Runtime;
+using Microsoft.Crank.EventSources;
+
+namespace HttpClientBenchmarks;
+
+class Program
+{
+    private static ClientOptions _options = null!;
+    private static List<HttpClient> _httpClients = new();
+    private static ConcurrentBag<long>? _headersTimes;
```

Comment:
Do you also intend to accept sending custom headers? (e.g., `-H` multiple times)

Reply:
You mean passing them from the outside of the app? We might want to add that in the future as low pri, but in the beginning we were thinking of just hard-coding/generating them inside a scenario, possibly based on some parameter.

Reply:
It's definitely interesting but doesn't need to be included in this PR.